mathics/core/pattern.py

```diff
@@ -532,19 +486,24 @@ def get_match_count(self, vars_dict: Optional[dict] = None):
         """the number of matches"""
         return (1, 1)
 
-    def get_wrappings(
-        self,
-        yield_func: Callable,
-        items: Tuple,
-        max_count: Optional[int],
-        expression: Expression,
-        attributes: int,
-        include_flattened: bool = True,
-    ):
-        """Get the possible wrappings"""
+    def get_wrappings(self, yield_func: Callable, items: Tuple, pattern_context: dict):
+        """
+        Get the possible wrappings
```

Comment:
I don't understand and possibly don't like the term "wrappings" which is vague. 

What exactly _is_ a wrapping?  Changing the name can be left for the future though.

Reply:
To be honest, I neither fully understand/like this term. What is true is that this function is called a lot, and seems to be related to the different orders/ grouping that the same expression must be (tried to) match against a pattern. The docstring has the best understand I could get from looking at the code.

Reply:
ok.